services/thermal_mgr/thermal_mgr.c

```diff
@@ -43,18 +43,38 @@ void initThermalSystemManager(lm75bd_config_t *config) {
 
 error_code_t thermalMgrSendEvent(thermal_mgr_event_t *event) {
```

Comment:
Check if `event == NULL` before using it. If it's null, return ERR_CODE_INVALID_ARG

Reply:
Done, will push change.